### DISCUSSION TITLE
TP: Diferencia entre usuario y organización

### DISCUSSION THREAD

**Discussion by resLotus** [dds-jv/foro#122] [TPA - 2020] [answered]
Hola, quería consultar una duda sobre el TP. 
Asumimos que el usuario y la organización son entidades separadas, esto es así? O existe la posibilidad de que el usuario sea la organización en si?
Y como serian las relaciones entre usuarios y organizaciones? Podría un usuario tener varias organizaciones o una organización varios usuarios?

**Comment by roli93** [comment]
Hola! En esta iteración por lo pronto no se vislumbra ningún tipo de relación evidente entre los usuarios y las organizaciones.

**Comment by gastonprieto** [comment]
@resLotus te sirvio?

**Comment by resLotus** [comment, marked as answer]
Hola, 
sisisi nos fue util
Disculpen la demora en contestar, vi el mail y se me paso
